### PR TITLE
Update ods to 6.0.0

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Update ODS to v6.0.0 and adapt CSS color variables & docs to the changes

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14814/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14814/

<p>















</p></details>

File: docs/theming/_index.md [hunk 1]
```diff
@@ -125,55 +133,75 @@ Again, you can use the [ownCloud design tokens](https://owncloud.design/#/Design
...
+### Sizes
+
+You can change the `form-check-default` (checkbox sizing), `height-small` (height of the logo img in the sidebar), `icon-default` (icon base size) and `width-medium` (width of the sidebar) according to your needs. If you need more customization options regarding sizes, please [open an issue on GitHub](https://github.com/owncloud/web/issues/new) with a detailed description.
+
+Size variables get prepended with `--oc-size-`, so e.g. *"icon-default"* creates the custom CSS property `--oc-size-icon-default`.
+
 ### Spacing
 
-Changing the spacing via a custom theme is not yet supported and will be released in the future. If this is a pressing feature you'd like to use soon, please [open an issue on GitHub](https://github.com/owncloud/web/issues/new) or reach out to us about contribution options.
+Use the six spacing options (`large | medium | small | xlarge | xsmall | xxlarge`) to create a more (or less) condensed version of the user interface. If you need more customization options regarding sizes, please [open an issue on GitHub](https://github.com/owncloud/web/issues/new) with a detailed description.
```
Comment (kulmann) [inline docs/theming/_index.md:202]:
```suggestion
Use the six spacing options (`xsmall | small | medium | large | xlarge | xxlarge`) to create a more (or less) condensed version of the user interface. If you need more customization options regarding sizes, please [open an issue on GitHub](https://github.com/owncloud/web/issues/new) with a detailed description.
```
I'd prefer to sort the spacing options in ascending order.

**Review by kulmann — CHANGES_REQUESTED** [review]
There was a time before we had an [ocis monorepo](https://github.com/owncloud/ocis). We had an `ocis-web` repo back then, which is now the `web` folder within the ocis monorepo. Having that in mind, it's now easier to release web. 1) release web, 2) then update web in ocis, 3) be done.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14816/

<p>







</p></details>

**Review by kulmann — APPROVED** [review]

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14817/

<p>









</p></details>

**Pull request merged**